codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/rustlang/RustTypes.kt

```diff
@@ -364,4 +381,4 @@ sealed class Attribute {
     }
 }
 
-data class Argument(val argument: String, val value: String)
+data class Argument(val argument: String, val value: String, val type: String)
```

Comment:
I think an argument could be made to store the `RustType` rather than a `String` for `type`, but it's not needed right now and not blocking.

Reply:
I didn't do that because I wasn't sure how to represent `into Impl` as a rust type